web/src/app/components/cdf/cdf.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { CdfComponent } from './cdf.component';
+
+describe('CdfComponent', () => {
```

Comment:
s/CdfComponent/CDFComponent
CDF is already the short form, so we should capitalize them.
Same for other places.